xbmc/dbwrappers/DatabaseQuery.cpp

```diff
@@ -246,7 +246,7 @@ std::string CDatabaseQueryRule::FormatParameter(const std::string &operatorStrin
     parameter = " IN (" + parameter + ")";
   }
   else
-    parameter = db.PrepareSQL(operatorString.c_str(), ValidateParameter(param).c_str());
+    parameter = db.PrepareSQL(operatorString, ValidateParameter(param));
```

Comment:
this is a C wrapper, so likely needs c_str

Reply:
https://github.com/xbmc/xbmc/blob/a644f0674c06cb6080612ad9c9d8b0d5a9f8a41a/xbmc/dbwrappers/Database.h#L114

Reply:
Ah, va_args, meh, the first one can still be removed though.